### PR TITLE
fix(#783) - Feedback SPA fixes

### PR DESCRIPTION
**Pull request by hybridx**
# Closes/Fixes/Resolves

# Explain the feature/fix

Added loader and UI fixes for the feedback-spa

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/feedback-spa/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --base-href /feedback/",
+    "build": "ng build --prod --base-href /feedback/",
```
Comment (ghanlohar) [inline packages/feedback-spa/package.json:7]:
```suggestion
    "build": "ng build --production --base-href /feedback/",
```

Comment (ghanlohar) [reply packages/feedback-spa/package.json:7]: @hybridx Should it be --production here in the build command?

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/feedback-spa/package.json (hunk 1 above)
Comment (hybridx) [reply packages/feedback-spa/package.json:7]: This is for the ng cli.
Here's the documentation link I referred `https://angular.io/cli/build`

Comment (riginoommen) [reply packages/feedback-spa/package.json:7]: This looks right

**Review by riginoommen — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]

**Pull request merged**